backend/src/services/auth/auth-login-service.ts

```diff
@@ -511,9 +537,9 @@ export const authLoginServiceFactory = ({
         isEmailVerified: user.isEmailVerified,
         firstName: user.firstName,
         lastName: user.lastName,
+        hasExchangedPrivateKey: Boolean(userEnc?.serverEncryptedPrivateKey),
```

Comment:
Oh nice, so as long as they go through the normal flow at least once, they will not have to enter in master password for oauth flow?